config/app.php

```diff
@@ -13,7 +13,7 @@
     |
     */
 
-    'name' => env('APP_NAME', 'Laravel'),
+    'name' => env('APP_NAME', 'Blog'),
```

Comment:
Debería de ser Blog Api
```suggestion
    'name' => env('APP_NAME', 'Blog Api'),
```